raiden-cli/src/routes/userDeposit.ts

```diff
@@ -0,0 +1,151 @@
+import { Zero } from '@ethersproject/constants';
+import { BigNumber } from 'ethers';
+import type { Request, Response } from 'express';
+import { Router } from 'express';
+
+import type { Cli } from '../types';
+import {
+  isConflictError,
+  isInsuficientFundsError,
+  isInvalidParameterError,
+} from '../utils/validation';
+
+type DepositResponse = { transaction_hash: string };
+type PlanWithdrawResponse = { transaction_hash: string; planned_withdraw_block_number: number };
+type WithdrawResponse = { transaction_hash: string };
```

Comment:
Please, for interfaces we control (i.e. are not enforced by external APIs), let´s keep consistent variable and members naming and use camelCase for those; we may re-introduce the linting rule later on, but for now we just need to keep an eye on this

Reply:
Scratch that, should have finished reading the file before submitting this one, it´s obviously enforced by the external reference.